src/stores/authenticationStore.ts

```diff
@@ -25,6 +29,14 @@ export const useAuthenticationStore = defineStore('authentication', {
       return Vue.prototype.$store.state.account?.currentAccount
     },
 
+    /**
+     * The (Keycloak) current account id.
+     * @remarks See getCurrentAccount remarks.
+     */
+    getCurrentAccountId (): number {
+      return this.getCurrentAccount?.id
```

Comment:
We fetch current account on line 29 directly from the state/account store (shared with SBC Header which actually does the login/logout).